src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -5786,38 +5786,53 @@ public void disableAuditLog()
         logger.info("Auditlog is disabled");
     }
 
+    @Deprecated
```

Comment:
nit: I believe you forgot the corresponding @Deprecated in `StorageServiceMBean`.  This can be done on commit

Reply:
ah right ...